### PR TITLE
Remove href="#" from html A tags to avoid unwanted redirections

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Using href="#" causes a redirect when  html5mode is on.
Using href="" causes a redirect when html5mode is off.
Removing it completely doesn't cause a redirect in either case.

### CODE REVIEW

**Comment by waldenraines** [comment]
Will the `<a>`s get `cursor:pointer;` without the `href`?

**Comment by dtaylor113** [comment]
LGTM

**Comment by jeff-phillips-18** [comment]
Ugh, @waldenraines you are correct, we lost the pointer. I will fix it.

**Comment by jeff-phillips-18** [comment]
Add CSS to put back missing pointer cursors

File: styles/angular-patternfly.css [hunk 1]
```diff
@@ -413,3 +413,11 @@
   border-color: #0076b7 !important;
   color: #fff !important;
 }
+
+.toolbar-pf-actions .toolbar-pf-view-selector a {
+  cursor: pointer;
+}
+
+.toolbar-pf-actions .dropdown-menu a {
+  cursor: pointer;
```
Comment (waldenraines) [inline styles/angular-patternfly.css]: Thoughts about making an `<a>` always have `cursor: pointer;` not just within the toolbar?

**Comment by jeff-phillips-18** [comment]
I'm hesitant to make that broad of a stroke.

**Comment by waldenraines** [comment]
> I'm hesitant to make that broad of a stroke.

Yeah, I hear ya, but I wonder if there is any valid reason to use an `<a>` that is not a link.

I won't hold this up on this issue but it may be something to revisit if we find ourselves having to do this elsewhere.

**Comment by waldenraines** [comment]
@jeff-phillips-18 this will need a rebase now because I just merged #100, sorry about that.

**Comment by jeff-phillips-18** [comment]
Rebased and squashed

**Pull request merged**